base/providers.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region     = local.region
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
```

Comment:
Thanks for your sharing 🙏🏻 I got the idea now.

Since I need to provide the `access key` and the `secret key` many times while planning the resources, then I've already set up the `variables set` in the Terraform Cloud 🙏🏻 

<img width="1775" alt="image" src="https://github.com/sanG-github/nimble-devops-ic-infrastructure/assets/63148598/d9028fa7-5c72-468f-bcb3-2b3fe1826f95">